web/config.py

```diff
@@ -0,0 +1,4 @@
+db_file = "database.db"
```

Comment:
If constant names be capitalized would be better

Reply:
sure will do